packages/http-client-java/generator/http-client-generator-clientcore-test/Generate.ps1

```diff
@@ -84,6 +84,14 @@ $job | Receive-Job
 
 Remove-Item ./specs -Recurse -Force
 
+git fetch origin pull/6981/head:smoke-test-branch
+git restore --source smoke-test-branch --worktree -- ../../../smoke-http-specs
+Copy-Item -Path ../../../smoke-http-specs/specs -Destination ./ -Recurse -Force
```

Comment:
Script here is just copy the 2 tsp files into local.

This is probably much simpler that one commit the tsp into this branch.